R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -243,7 +243,7 @@ makeTreeSummarizedExperimentFromBiom <- function(obj, ...){
     col = x[ , colname]
     # List prefixes
     prefixes <- c(
-        "^d__",
+        "^sk__",
```

Comment:
Just add sk__, don't remove d__

Reply:
Ahh, now I know why you have done this

Hmmm, I think solution is to replace TAXONOMY_RANKS with your own variable taxonomy_ranks <- c("domain", "superkingdom", "kingdom"...)

That is because now user can modify TAXONOMY_RANKS, but those prefixes are still hardcoded

Reply:
Or maybe better

prefixes <- c(domain = "d__", superkingdom = "sk__"...)

And based on matches, get the corresponding name